Makefile

```diff
@@ -3,8 +3,8 @@ PWD := $(shell pwd)
 test:
 	./check-fmt.sh
 	bazel version
-	bazel build //... --action_env=GIT_ROOT=$(PWD) --sandbox_writable_path=$(PWD) --incompatible_remove_native_http_archive=false
-	bazel test --test_output=errors //... --action_env=GIT_ROOT=$(PWD) --sandbox_writable_path=$(PWD) --incompatible_remove_native_http_archive=false
+	bazel build //... --action_env=GIT_ROOT=$(PWD) --sandbox_writable_path=$(PWD) --incompatible_remove_native_http_archive=false --sandbox_debug
```

Comment:
what's this for?

Reply:
just for debugging, the build is failing and I wanna know why. it'll get removed before merge.